### PR TITLE
#169 - Add IT, try to reproduce regex bug

### PR DESCRIPTION
**Pull request by genryxy**
Part of #169 
Add IT for installation published package using `npm publish` or `curl PUT`. These ITs work well

### CODE REVIEW

File: src/test/java/com/artipie/npm/NpmIT.java [hunk 1]
```diff
@@ -154,6 +159,26 @@ void npmInstallWorks() throws Exception {
         );
     }
 
+    @Test
+    void installsPublishedProject() throws Exception {
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/npm/NpmIT.java:163]: @genryxy I'd suggest to parameterize this test case as the previous one

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy thanks, one suggestion

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Review by olenagerasimova — DISMISSED** [review]

**Comment by olenagerasimova** [comment]
@g4s8 please take a look/merge

**Comment by genryxy** [comment]
@olenagerasimova could you please take a look again because I made changes about naming which were supposed in another related PR

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, I've accidentally pressed the wrong button in that PR

**Comment by olenagerasimova** [comment]
@g4s8 please, take a look/merge

**Review by g4s8 — APPROVED** [review]
@genryxy thanks

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#195` is not assigned, can't get performer

**Pull request merged**